plugins/nginx-vhosts/commands

```diff
@@ -49,6 +49,9 @@ case "$1" in
       shopt -u nullglob
     fi
 
+    DOKKU_APP_CONTAINER_ID=$(< "$DOKKU_ROOT/$APP/CONTAINER")
+    docker cp "$DOKKU_APP_CONTAINER_ID:/app/nginx.conf" "$APP_NGINX_TEMPLATE" 2> /dev/null || true
```

Comment:
Yes but that's not how to get the container id anymore since we did scaling. You'll want something like [this](https://github.com/progrium/dokku/blob/483ca9627f390a2b5c7e0382488cad95fc62b032/plugins/00_dokku-standard/commands#L120). Then grab the first one or just use the image and start a container with `--rm` maybe?